### PR TITLE
getRepositoriesData selector in PullRequestSummary's...

### PR DESCRIPTION
**Pull request by takarzyna**
...RepositoriesSection - unit test

### CODE REVIEW

File: src/client/containers/PullRequestSummary/RepositoriesSection/__tests__/index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import chai from 'chai'
+import {
+  getRepositoriesData,
+} from '../index'
+
+const expect = chai.expect
+
+describe('getRepositoriesData', () => {
```
Comment (kkateq) [inline src/client/containers/PullRequestSummary/RepositoriesSection/__tests__/index.js]: Please name it properly and add test for negative cases.
